docs/coturn.md

```diff
@@ -0,0 +1,155 @@
+    ```
+    ls -l /lib/systemd/system/coturn.service
+    ```
+
+- If you installed coTURN manually, you may want to create a sysvinit service or systemd unit, or use another method to run the following during boot:
+    ```
+    /path/to/turnserver -c /path/to/turnserver.conf -o
+    ```
+
+- `-o` starts the server in daemon mode, `-c` defines the path to the config file.
+- There is also an official example available at [https://github.com/coturn/coturn/blob/master/examples/etc/coturn.service](https://github.com/coturn/coturn/blob/master/examples/etc/coturn.service)
+
+##### Running coTURN on privileged ports
+
+On some GNU/Linux distributions (for example, **Ubuntu Focal and later**) when _coTURN_ is installed from the official package the _coturn_ service is executed as an unprivileged user like _turnserver_. Due to this by default _coTURN_ can not use privileged ports, like port 443.
+
+Depending on the system configuration Linux kernel capabilities could be used to overcome this limitation. Capabilities can be associated with executable files using _setcap_, so you could allow the _/usr/bin/turnserver_ executable to bind sockets to privileged ports with:
+```
+setcap cap_net_bind_service=+ep /usr/bin/turnserver
+```
+
+Alternatively, if the system configuration does not allow to set the capability, or if the coturn process needs to access files only readable by root like an SSL certificate for TLS connections, you could configure the _coturn_ service to be executed by root instead of the unprivileged user by executing:
+```
+systemctl edit coturn
+```
+and then setting the following configuration, which will override the default one:
+```
+[Service]
+User=root
+Group=root
+```
+
+#### 3. Configure `turnserver.conf` for usage with Nextcloud Talk
+
+- Next you need to adjust the coTURN configuration file to work with Nextcloud Talk.
+- Choose the listening port (default is _3478_) and an authentication secret, where a random hex is recommended
+    ```
+    openssl rand -hex 32
+    ```
+
+- Then uncomment/edit the following settings accordingly:
+
+```
+listening-port=<yourChosenPortNumber>
+fingerprint
+lt-cred-mech # Only on coTURN below v4.5.0.8!
+use-auth-secret
+static-auth-secret=<yourChosen/GeneratedSecret>
+realm=your.domain.org
+total-quota=0
+bps-capacity=0
+stale-nonce
+no-loopback-peers # Only on coTURN below v4.5.1.0!
+no-multicast-peers
+```
+
+- Support for TLS connections to the TURN server has been added in Talk 11.
+
+  > In some cases clients can be behind very restrictive firewalls that only allow TLS connections; in those cases the clients would be able to connect with other clients or the High Performance Backend only through a TURN server and a TLS connection. However, please note that TLS connections do not provide any additional security, as media streams are always end-to-end* encrypted in WebRTC; enabling TLS is just a matter of providing the maximum compatibility.
```

Comment:
while this fixes the indentation, it adds a blockquote to the section which is just not rendered visually at the moment.